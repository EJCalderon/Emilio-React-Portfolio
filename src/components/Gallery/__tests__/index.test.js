import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Gallery from '..'
const portrait = { name: "portraits", description: "people"};

afterEach(cleanup)

describe('Gallery component', () => {

    it('renders', () => {
        render(<Gallery currentCategory={portrait} />);
    });

    it('mathces snapshot', () => {
        const { asFragment } = render(<Gallery currentCategory={portrait}/>)
        expect(asFragment()).toMatchSnapshot()
    })
});

it('displays text', () => {
    const { asFragment } =render(<Gallery  currentCategory={portrait} />)
    expect(asFragment()).toMatchSnapshot()
})
