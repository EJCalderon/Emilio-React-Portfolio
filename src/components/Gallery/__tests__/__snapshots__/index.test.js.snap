// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gallery component mathces snapshot 1`] = `
<DocumentFragment>
  <section>
    <h1
      data-testid="h1tag"
    >
      Portraits
    </h1>
    <p>
      people
    </p>
    <div>
      <div
        class="flex-row"
      />
    </div>
  </section>
</DocumentFragment>
`;

exports[`displays text 1`] = `
<DocumentFragment>
  <section>
    <h1
      data-testid="h1tag"
    >
      Portraits
    </h1>
    <p>
      people
    </p>
    <div>
      <div
        class="flex-row"
      />
    </div>
  </section>
</DocumentFragment>
`;
